src/tools/MagnifyTool.js

```diff
@@ -3,6 +3,7 @@ import { getNewContext } from '../drawing/index.js';
 import BaseTool from './base/BaseTool.js';
 import { hideToolCursor, setToolCursor } from '../store/setToolCursor.js';
 import { magnifyCursor } from './cursors/index.js';
+import { isDebuggerStatement } from '@babel/types';
```

Comment:
What is this?

Reply:
My IDE making bad guesses about my life decisions.